app/views/homepages/index.html.erb

```diff
@@ -0,0 +1,52 @@
+<h2><span class="spotlight-prefix">Media Spotlight:</span> <%= link_to Work.media_spotlight.title, work_path(Work.media_spotlight.id), class: "spotlight-link" %> <span class="spotlight-author"> by <%= Work.media_spotlight.creator %></span></h2>
+<p><%= Work.media_spotlight.vote_ids.length %> votes - <%= Work.media_spotlight.description %></p>
+
+<hr class="root-hr">
+
+<section class="top-ten-container">
+  <section class="top-ten__list-container">
+    <h2 class="top-ten__header">Top Movies</h2>
+    <% Work.top_ten("movie").each do |work| %>
```

Comment:
You have the same code to show a list of works repeated 3 times. Could you use a view partial or a loop to DRY this up?